ts/Dashboards/EditMode/EditMode.ts

```diff
@@ -558,6 +560,8 @@ class EditMode {
 
         editMode.active = true;
         editMode.isContextDetectionActive = true;
+
+        eventsOptions?.activate.call(editMode);
```

Comment:
When enabling edit mode, an error is thrown on this line and edit mode does not activate. `eventsOptions.activate` seems to be undefined; adding optional chaining to `activate` does not solve the problem.
![image](https://github.com/highcharts/highcharts/assets/46498046/9e0beb0e-1bf4-4579-950d-bbc7dcf34c93)

Reply:
By mistake, I set `activated` instead of `activate` in the demo.

Thanks!